apps/parse-rated-paragraphs/src/app/parse-rated.ts

```diff
@@ -2,12 +2,20 @@ import * as fs from 'fs';
 import {
   ParsedArticleParagraphStandalone,
   ArticleParagraphBacksUpClaim,
+  ImpactFileList,
 } from '@foodmedicine/interfaces';
 
 const allParagraphsBasePath = './tmp/correlated-paragraphs/';
```

Comment:
I believe Casey mentioned this earlier, *primitive consts* can be named in ALL_CAPS_UNDERSCORE style (so const strings, ints, etc). That way, when you're reading the code and see some constant in that ALL_CAPS style, you know its a const primitive that was defined earlier.

Reply:
Ok sure, changed